modules/nvidia_plugin/tests/functional/core_config.cpp

```diff
@@ -3,5 +3,14 @@
 //
 
 #include "functional_test_utils/core_config.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
 
 void CoreConfiguration(LayerTestsUtils::LayerTestsCommon* test) {}
+
+namespace ov {
+namespace test {
+
+void core_configuration(ov::test::SubgraphBaseTest* test) {}
```

Comment:
Please refer to https://github.com/openvinotoolkit/openvino_contrib/blob/3dfb560f8dfcbdc4642efa5a46a5f7128d638ef2/modules/nvidia_plugin/tests/functional/core_config.cpp from PR https://github.com/openvinotoolkit/openvino_contrib/pull/656

Reply:
@nkogteva Please check